gaia-fast-list.js

```diff
@@ -1311,57 +1251,75 @@ Internal.prototype = {
   },
 
   /**
+   * Set the scroll position of the list.
+   *
+   * It is common for users to want to set
+   * an initial scrollTop before the real
+   * list has actually rendered.
+   *
+   * To support this we transform the list element
+   * and unset the transform once rendered.
+   * This is to avoid reflowing an expensive
+   * component.
+   *
+   * @param {Number} value
+   */
+  setScrollTop(value) {
+    debug('set scroll top', value);
+    if (this.fastList) this.fastList.scrollInstantly(value);
+    else {
+      this.els.list.style.transform = `translateY(${-value}px)`;
```

Comment:
Yeah this is kinda funky. I need to fake the `scrollTop` when we don't actually have a scrollable list yet :/